packages/client/hmi-client/src/services/models/funman-service.ts

```diff
@@ -164,37 +164,89 @@ export const renderFumanTrajectories = (
 	element: HTMLElement,
 	processedData: FunmanProcessedData,
 	boxId: string,
-	options: RenderOptions
+	options: RenderOptions,
+	state: string
 ) => {
```

Comment:
From an API design perspective, you generally want to have things like "options" or "kwargs" as the last parameter. Order matters:
- Less error prone because options are some times equates with optional, so you should not have parameters afterward
- Similarly, mandatory parameters should be specified before optional parameters for consistency

Reply:
Makes sense I will throw options as last param instead of state